lms/djangoapps/discussion/api/external_discussions.py

```diff
@@ -0,0 +1,44 @@
+from opaque_keys.edx.keys import CourseKey, UsageKey
+
+from .data import ExternalDiscussionData
+from ..models import ExternalDiscussionsIdMapping
+
+
+def get_external_discussion_context(context_key: CourseKey, usage_key: UsageKey) -> ExternalDiscussionData:
```

Comment:
I think a context might include more data. Let's just call this:
```suggestion
def get_external_discussion_id(context_key: CourseKey, usage_key: UsageKey) -> ExternalDiscussionData:
```